src/masoniteorm/query/grammars/MSSQLGrammar.py

```diff
@@ -131,7 +131,7 @@ def table_string(self):
         return "[{prefix}{table}]"
 
     def order_by_string(self):
-        return "ORDER BY {column} {direction}"
```

Comment:
These need to go back

Reply:
Hmmm 
Yeah I understand the Grammar needs to be abstracted.
Unfortunately if the order_by_string is reverted  you end up with a query that looks something like this in the current 0.9.3 version

SELECT ..... ORDER BY "name" DESCORDER BY "age" ASC .....  

Which is obviously broken and why I moved the prefix in the first portion of the string.


That said to keep the Grammar abstracted how about something like this in the BaseGrammar class...  a class var 

order_by_prefix = "ORDER BY"

This would allow easy future abstraction while still being able to be uses in the process_order_by() like so

sql = self.order_by_prefix + sql

WDYT?

Reply:
Yeah I think that's fine. I don't really see what the difference is between prefixing it as a class property and putting it in these format string

We can do this: `sql = self.order_by_prefix + sql` and then maybe we can refactor later. I think the way I did some of the platform class coding is better for this situation anyway. Its a bit newer code and it handled this differently

Reply:
To see the difference use the test for multiple order by against the current 0.9 version and it will fail and you will see the reason why the format string will not work when it is used in this manner. 

Anyway I will update this PR in the morning.
Cheers

Reply:
@josephmancuso 
After thinking about this, looking at the existing format strings and keep the abstraction in place I have changed things slightly.

This adjustment now keeps formatting and keyword usage in their own class.
Let me know if there are  any pther considerations?